aries-site/src/examples/components/layer/LayerCenterExample.js

```diff
@@ -25,18 +33,21 @@ export const LayerCenterExample = () => {
                 <Box justify="center">
                   <MailOption />
                 </Box>
-                <Heading margin="none" level={2}>
+                <Heading margin="none" size="small" level={2}>
                   Modal Dialog
                 </Heading>
               </Box>
               <Box justify="center">
                 <Button icon={<FormClose />} onClick={onClose} />
               </Box>
             </Box>
+            <Paragraph margin="none">A subtitle if needed</Paragraph>
```

Comment:
This seems like it should be Text not Paragraph and the copy below, Paragraph, not Text.